src/components/Text/text/text.ts

```diff
@@ -0,0 +1,87 @@
+import {block} from '../../utils/cn';
+
+import './text.scss';
+
+const b = block('text');
+
+export const TYPOGRAPHY_VARIANTS = [
```

Comment:
`TEXT_VARIANTS` is better imo, it shows connection to `Text` component like `TEXT_COLORS` does.